src/main/java/frc/team4373/robot/commands/wheelspinner/ColorSpinCommandGroup.java

```diff
@@ -0,0 +1,17 @@
+package frc.team4373.robot.commands.wheelspinner;
+
+import edu.wpi.first.wpilibj.command.CommandGroup;
+import edu.wpi.first.wpilibj.command.WaitCommand;
+import frc.team4373.robot.RobotMap;
+
+public class ColorSpinCommandGroup extends CommandGroup {
+    /**
+     * Constructs a new command group for deploying the wheel spinner and spinning
+     * the wheel spinner to a color.
+     */
+    public ColorSpinCommandGroup() {
+        addSequential(new DeploySpinnerCommand());
+        addSequential(new WheelSpinnerColorCommand());
+        addSequential(new RetractSpinnerCommand());
```

Comment:
We will probably need to drive forward after deploying the spinner before spinning, and drive backward before retracting it.